runtime/interpreter/accountinbox.go

```diff
@@ -28,7 +28,7 @@ import (
 // AuthAccountInbox
 
 var authAccountInboxTypeID = sema.AuthAccountInboxType.ID()
-var authAccountInboxStaticType StaticType = PrimitiveStaticTypeAuthAccountKeys
+var authAccountInboxStaticType StaticType = PrimitiveStaticTypeAuthAccountInbox
```

Comment:
Oops! Thanks for catching this

Reply:
AAAA that's where my failing test was coming from, I knew it stemmed from static type resolution *somewhere*